pkg/services/auth/model.go

```diff
@@ -23,3 +20,51 @@ type userAuthToken struct {
 	UpdatedAt     int64
 	UnhashedToken string `xorm:"-"`
 }
+
+func userAuthTokenFromUserToken(ut *models.UserToken) *userAuthToken {
+	var uat userAuthToken
+	uat.fromUserToken(ut)
+	return &uat
+}
+
+func (uat *userAuthToken) fromUserToken(ut *models.UserToken) error {
```

Comment:
I think this is noisy. Since we couldn't isolate this struct completely I think we should use `models.UserToken` in all places.

Reply:
Not a blocker. So I'm fine merging this first and fixing it later.

Reply:
Somewhat agree that it's noisy, but still internal to the package I think it's okay. One of the nice things is that the database model is internal.